reVX/least_cost_xmission/layers/transmission_layer_io_handler.py

```diff
@@ -240,18 +190,21 @@ def load_tiff(self, fname: str, band: int = 1,
             return data
 
         if not reproject:
-            if data.shape != self.shape:
-                raise ValueError(f'Shape of {full_fname} ({data.shape}) '
-                                 'does not match template raster shape '
-                                 f'({self.shape}).')
-            if transform != self.profile['transform']:
-                raise ValueError(f'Transform of {full_fname}:\n{transform}\n'
-                                 'does not match template raster shape:\n'
-                                 f'{self.profile["transform"]}')
-            if crs != self.profile['crs']:
-                raise ValueError(f'CRS of {full_fname}:\n{crs}\ndoes not '
-                                 'match template raster shape:\n'
-                                 f'{self.profile["crs"]}')
+            pass
```

Comment:
Yea sorry, still actively working on this PR. I don't think I can convert it to a draft since it's your PR but I can request a review from you once it's ready.

It's going to be a big boy PR again lol

Reply:
P.S. TO actually answer your question - this part of the code was erroring out when it shouldn't be, so I am going to replace this with the more nuanced checks we had in `ExclusionsConverter` (which is now called `LayeredH5`)